src/installer/lime3ds.nsi

```diff
@@ -0,0 +1,201 @@
+; Copyright Dolphin Emulator Project
```

Comment:
Should this be removed or changed? I know most of this code comes from there, but I assume parts are different.

Reply:
On second look, it should probably also mention Lime3DS as well. A lot of the file was changed.